crypto/armor.go

```diff
@@ -254,36 +263,3 @@ func decryptPrivKey(saltBytes, encBytes []byte, passphrase, kdf string) (privKey
 
 	return legacy.PrivKeyFromBytes(privKeyBytes)
 }
-
-//-----------------------------------------------------------------
-// encode/decode with armor
-
-func EncodeArmor(blockType string, headers map[string]string, data []byte) string {
-	buf := new(bytes.Buffer)
-	w, err := armor.Encode(buf, blockType, headers)
-	if err != nil {
-		panic(fmt.Errorf("could not encode ascii armor: %w", err))
-	}
-	_, err = w.Write(data)
-	if err != nil {
-		panic(fmt.Errorf("could not encode ascii armor: %w", err))
-	}
-	err = w.Close()
-	if err != nil {
-		panic(fmt.Errorf("could not encode ascii armor: %w", err))
-	}
-	return buf.String()
-}
-
-func DecodeArmor(armorStr string) (blockType string, headers map[string]string, data []byte, err error) {
-	buf := bytes.NewBufferString(armorStr)
-	block, err := armor.Decode(buf)
-	if err != nil {
-		return "", nil, nil, err
-	}
-	data, err = io.ReadAll(block.Body)
-	if err != nil {
-		return "", nil, nil, err
-	}
-	return block.Type, block.Header, data, nil
-}
```

Comment:
Same code in "github.com/cosmos/crypto/armor"

Reply:
is this needed in crypto? if we just migrated all of the things from sdk to that repo then i think you will be starting with tech debt right away. I dont think this is a good plan and the repo should start fresh with better standards.